.env.example

```diff
@@ -20,6 +20,8 @@ OAUTH_GOOGLE_SECRET_KEY=CANNOT_SEND_REAL_KEY_GOCSPX-dzd6PPjmj4fi4LDm_r2KgQXEv7ot
 
 # JWT
 SECRET_KEY=test
+JWT_ACCESS_TOKEN_EXPIRATION_TIME=15min
+JWT_REFRESH_TOKEN_EXPIRATION_TIME=7d
```

Comment:
I believe that this is not so confidential info to keep in .env. Can it be just conts values?

Reply:
Agreed